modules/Emails/vardefs.php

```diff
@@ -483,6 +483,16 @@
             'inline_edit' => false,
         ),
 
+        'inbound_email_id' => array(
```

Comment:
This field is not needed. The mailbox_id field already stores the id of the inbound email record.